### DISCUSSION TITLE
Blackhole not working anymore with MacOs 11.3

### DISCUSSION THREAD

**Discussion by zerothawg** [ExistentialAudio/BlackHole#379] [Q&A]
After yesterday's MacOs Update to 11.3 blackhole does not seem to work anymore. Anyone else with the same issue? Just has disappeared from "Audio Devices".

**Comment by devinroth** [comment]
Try just reinstalling.

Devin Roth

**Reply by zerothawg** [reply]
ty! this fixed it!
